<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DT-TECH-2D80" ignored="1" name="test1" tests="330" failures="12" timestamp="07 Mar 2018 10:22:29 GMT" time="345.479" errors="0">
  <testcase name="loginTest" time="1.628" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.03" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.051" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.036" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.106" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.026" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.016" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.009" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.053" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.045" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.037" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.044" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.054" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.051" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.056" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.034" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.054" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.05" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.063" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.045" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.054" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.053" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.047" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.016" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.064" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckDE1"/>
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckDE1">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseDE1(AppKeywords.java:534)at testKeywords.Keyword.executeKeywords(Keyword.java:228)at testCases.DatabaseCheckDE1.loginTest(DatabaseCheckDE1.java:39)at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.044" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.064" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.041" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.158" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.267" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.038" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.049" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.158" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.064" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.035" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.079" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.158" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.051" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.045" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.018" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.048" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.032" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.033" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.018" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.016" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.018" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.04" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.017" classname="testCases.DatabaseCheckChecker"/>
  <testcase name="loginTest" time="1.0" classname="testCases.DatabaseCheckChecker">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1at testKeywords.AppKeywords.SelectDatabaseChecker(AppKeywords.java:630)at testKeywords.Keyword.executeKeywords(Keyword.java:252)at testCases.DatabaseCheckChecker.loginTest(DatabaseCheckChecker.java:39)at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="@AfterTest quit" time="0.0" classname="testBase.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterTest quit -->
</testsuite> <!-- test1 -->
